corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -238,7 +238,7 @@ IntEntry.prototype.onPreProcess = function (newValue) {
 function PhoneEntry(question, options) {
     FreeTextEntry.call(this, question, options);
     this.templateType = 'str';
-    this.lengthLimit = options.lengthLimit || 15;
+    this.lengthLimit = options.lengthLimit;
```

Comment:
Oh, so this was already set up to work when `options.lengthLimit` is falsey (`undefined`? `null`? `0`? Not sure what the value is)

Reply:
yes. i tried it out and when it's undefined / null, it still works